.github/workflows/schedule_tests.yml

```diff
@@ -117,8 +117,6 @@ jobs:
       - name: Install and upgrade packaging tools
         run: python -m pip install --upgrade pip setuptools wheel
       - run: python -m pip install -r tests/requirements/py3.txt -r tests/requirements/postgres.txt -e .
-      - name: Create PostgreSQL settings file
-        run: mv ./.github/workflows/data/test_postgres.py.tpl ./tests/test_postgres.py
```

Comment:
I decided to try persisting the `test_postgres.py` file in `tests`, and now have seen that Jenkins is actually pulling in its own `test_postgres.py` file. 

Is the Jenkins config publicly available anywhere? I can see the steps it takes in test output, but if it's possible, it feels like we could have something cleaner here with environment variables rather than having jenkins write out a settings file.

Reply:
Please revert these changes. `schedule_tests` and `selenium` have nothing to do with Jenkins configuration.

> Is the Jenkins config publicly available anywhere?

No, it's not. I'm not sure what would like to change there.

Reply:
Sorry, I wasn't clear. Basically there is no Postgres settings file for tests checked in. Instead GitHub Actions copy over this template file (that has no template files), and Jenkins on its side copies over a file from ... somewhere (I can see this in the test log output of a CI job). search for test_postgres.py in test log output for a Postgres suite and you'll see it.

I do not know what the Jenkins settings file looks like though. Is it the same as this template?

In any case I'll revert that change here and start a separate ticket on that.

Reply:
> search for test_postgres.py in test log output for a Postgres suite and you'll see it.

I know how it's configured :smile: 

> I do not know what the Jenkins settings file looks like though.

There is nothing in the CI settings that would affect this patch. I wonder why you think it's important for your PR :thinking: 

> Is it the same as this template?

Almost the same, the only difference is that `DEFAULT_TABLESPACE` and `DEFAULT_INDEX_TABLESPACE` are set.

Reply:
Short version is I was trying to debug a postgres test failure I saw in CI and thought it the opportunity to bring up the point.

Anyways, [here's](https://code.djangoproject.com/ticket/35923#ticket) a Trac ticket for this, and going to revert the change on this branch to avoid the side quest.